lib/resolvers/parse/ast/fromObject.js

```diff
@@ -32,6 +34,10 @@ const _variableToValue = variable => {
     return _arrayToListValue(variable)
   } else if (isPlainObject(variable)) {
     return _objectToObjectValue(variable)
+  } else if (variable === null) {
+    return _nullValue
+  } else if (variable === undefined) {
```

Comment:
Can that really happen?

Reply:
Yes, suppose this is the query (same as in the following payloads, but formatted nicely):
```graphql
query ($filter: [AdminService_Books_filter]) {
  AdminService {
    Books(filter: $filter) {
      nodes {
        title
      }
    }
  }
}
```

If the variable is explicitly set to null, such as in this payload:
```json
{
  "query": "query ($filter: [AdminService_Books_filter]) {\n  AdminService {\n    Books(filter: $filter) {\n      nodes {\n        title\n      }\n    }\n  }\n}",
  "variables": {
    "filter": null
  }
}
```
line 38 will be reached.

If the variable is omitted entirely, such as in this payload:
```json
{
  "query": "query ($filter: [AdminService_Books_filter]) {\n  AdminService {\n    Books(filter: $filter) {\n      nodes {\n        title\n      }\n    }\n  }\n}",
  "variables": {}
}
```
line 40 will be reached.